README.md

```diff
@@ -199,6 +199,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
 - `DeprecatedGuavaObjects`: `com.google.common.base.Objects` has been obviated by `java.util.Objects`.
 - `JavaTimeSystemDefaultTimeZone`: Avoid using the system default time zone.
 - `IncubatingMethod`: Prevents calling Conjure incubating APIs unless you explicitly opt-out of the check on a per-use or per-project basis.
+- `CompileTimeConstantViolatesLiskovSubstitution`: Requires consistent application of the `@CompileTimeConstant` annotation.
```

Comment:
I'd flesh this out to say something like "annotating a subclass method with `@CompileTimeConstant` but leaving a parent class/interface un-annotated may result in the validation being missed"

Reply:
:+1: